ksql-common/src/main/java/io/confluent/ksql/schema/ksql/LogicalSchema.java

```diff
@@ -161,14 +153,6 @@ public LogicalSchema withoutAlias() {
     return new LogicalSchema(builder.build());
   }
 
-  /**
-   * @return {@code true} is aliased, {@code false} otherwise.
-   */
-  public boolean isAliased() {
```

Comment:
This method was fundamentally broken as we no longer ensure a schema contains columns that are either _all_ aliased or not aliased.  We now allow a mix. Hence I've removed it.

Reply:
I feel like this is going a step back - I feel like logical schemas should not have aliases _anywhere_ let alone mixed aliases. That gets really confusing to reason about... I think you convinced me of that a while back! A `LogicalSchema` represents the schema for a single source, so of course they all have the same alias.

Maybe it makes sense for `LogicalSchema` to have a top level `SourceName` that refers to all columns.

Reply:
I think this is just a short term thing right? We're moving towards dropping the source name from the schema altogether, and resolving qualifiers in expressions as part of analysis?

Reply:
Right, I'm just removing broken functionality.  There is follow up work needed to clear up the mixed-mode schemas and/or remove the source completely.  But that's not the intent of this PR. This PR is about keeping the logical schema free of the implementation details.